### ISSUE TITLE
Will this mod fix the "not waking up from sleep" issue as well?

### ISSUE THREAD

**Issue by Deppjones123** [stecker870/MacBook-Pro-15-Zoll-late-2011-Graphics-Card-Fix-#4]
As far as I understood, the mod brings back the brightness slider functionality for devices with High Sierra and disabled AMD dGPU. What the waking up from sleep when closing/opening the lid as well?

EDIT: I already found the answer, that a hardware mod is necessary to bring back the sleep function.

Another question about the brightness mod: When I use the mod in High Sierra, the OS starts with a low brightness setting for example. If I push either the F1 or F2 button only once, the brightness jumps to the maximum setting and I have to go down to the setting that I like. Is that a known issue? Bye the way, Mojave doesn't have this effect.

One more thing: Is it necessary for the brightness control to have the automatic switching of the graphic cards (on top of the energy settings) always enabled? I guess because of that, there is always a short graphics error during the boot process (when the apple is visible)

**Comment by stecker870** [comment]
> As far as I understood, the mod brings back the brightness slider functionality for devices with High Sierra and disabled AMD dGPU. What the waking up from sleep when closing/opening the lid as well?
> 
> EDIT: I already found the answer, that a hardware mod is necessary to bring back the sleep function.
> 
> Another question about the brightness mod: When I use the mod in High Sierra, the OS starts with a low brightness setting for example. If I push either the F1 or F2 button only once, the brightness jumps to the maximum setting and I have to go down to the setting that I like. Is that a known issue? Bye the way, Mojave doesn't have this effect.
> 
> One more thing: Is it necessary for the brightness control to have the automatic switching of the graphic cards (on top of the energy settings) always enabled? I guess because of that, there is always a short graphics error during the boot process (when the apple is visible)

yes everything works fine! it is only important to use the terminal for the nvram method! SSH or vnc in the mbp "nvram fa4ce28d-b62f-4c99-9cc3-6815686e30f9: gpu-power-prefs =% 0" reboot !!!
and in no case reset nvram ! 
you can find more here https://www.jeffgeerling.com/blog/2017/fixing-2011-macbook-pro-booting-grey-screen-amd-radeon-video-glitch
